examples/nextjs/pages/markets/[marketAccount]/place-bet.tsx

```diff
@@ -65,12 +74,11 @@ const PlaceBet = () => {
             if (!market) {
                 throw "Market is missing";
             }
-            const { backing, odds, stake } = formData[marketOutcome];
-            const marketTokenPk = new PublicKey(market.mintAccount);
+            const { forOrAgainst, odds, stake } = formData[marketOutcome];
+            const marketTokenPk = new PublicKey(market.account.mintAccount);
             const mintInfo = await getMintInfo(program, marketTokenPk);
             const stakeInteger = new BN(stake * 10 ** mintInfo.data.decimals);
-            const createBetOrderResposne = await createBetOrder(program, marketAccount, marketOutcomeIndex, backing === "back", odds, new BN(stakeInteger));
-            console.log('createBetOrderResposne', createBetOrderResposne);
+            const createBetOrderResposne = await createOrder(program, marketAccount, marketOutcomeIndex, forOrAgainst === "For", odds, new BN(stakeInteger));
```

Comment:
`Bet` is verboten. `createOrderResponse`. Is this being assigned just to appease the linter? I was being logged before but now I think it's just immediately dropped?

Reply:
also `Resposne` vs `Response`